### ISSUE TITLE
add flow typings

### ISSUE THREAD

**Issue by mweststrate** [mweststrate/immer#34]

**Comment by AriaFallah** [comment]
Wouldn't it be the same as typescript?

```js
function immer<T>(baseState: T, thunk: (draftState: T) => void): T
```

**Comment by mweststrate** [comment]
Probably yes 😊. Didn't figure out how flow should find the typings though.
So probably an easy PR for anybody familiar with that

**Comment by mweststrate** [comment]
Fixed by 0.8.4
